app/views/govuk_component/button.raw.html.erb

```diff
@@ -16,6 +17,10 @@
     href="<%= href.try(:html_safe) %>"
     role="button"
   <% end %>
+
+  <% if rel %>
+    rel="<%= rel.try(:html_safe) %>"
```

Comment:
What's the `html_safe` for?

Reply:
To be honest I was just following the convention for the previous passed arguments.

I think we could remove this since we're expecting plain text strings, I'll pop an update in, thanks Paul 👍